apps/web-mzima-client/src/app/settings/roles/role-item/role-item.component.ts

```diff
@@ -136,6 +134,7 @@ export class RoleItemComponent implements OnInit {
         },
       });
     } else {
+      delete roleBody.name; // We don't want to change name property on edit/update
```

Comment:
According to the discussion yesterday we will be requiring the name field to be intact in the update request and that it be unchanged from the original object.